src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/DeferredBuildTest.cs

```diff
@@ -0,0 +1,100 @@
+﻿using Microsoft.Build.Framework;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using Xamarin.ProjectTools;
+
+namespace Xamarin.Android.Build.Tests
+{
+    [Parallelizable(ParallelScope.Children)]
```

Comment:
Our code convention is tabs, not spaces.

Reply:
If there is no .editorconfig in the repo....

Reply:
Fixed by manually formatting...